eng/Versions.props

```diff
@@ -8,7 +8,7 @@
     <VersionMajor>7</VersionMajor>
     <VersionMinor>0</VersionMinor>
     <VersionSDKMinor>1</VersionSDKMinor>
```

Comment:
@marcpopMSFT shouldn't installer already be at 7.0.2xx? This will generate 7.0.102 installers.

Reply:
there were merge conflicts so GH was showing you the incoming.  I resolved the conflict by accepting the current state as we always do which should keep it on 7.0.200.

Reply:
also means no changes so I'll just close this